hoomd/md/EvaluatorPairMoliere.h

```diff
@@ -38,15 +38,76 @@ class EvaluatorPairMoliere
 {
     public:
         //! Define the parameter type used by this pair potential evaluator
-        typedef Scalar2 param_type;
+        struct param_type
+            {
+            // just holding on to these so if the user calls asDict,
+            // we can give them back correctly
+            unsigned int Zi;
+            unsigned int Zj;
+            Scalar e;
+            Scalar a0;
+
+            // these are the params the potential cares about
+            Scalar Zsq;
```

Comment:
It is not possible to reconstruct 4 values from 2. We should refactor this potential to accept different parameters so that it does not need to store extra information in a performance critical data structure.
* qi: Zi*e
* qj: Zj*e
* aF

The computation of aF from a0 should be moved to the documentation to show users how to correctly calculate the value.